<launch>
 <group>
   <rosparam command="load" file="$(find aria_utils)/settings/id_map.yaml" ns="/id_map"/>
   <rosparam command="load" file="$(find aria_utils)/settings/initial_pose.yaml" ns="/initial_pose"/>
   <rosparam command="load" file="$(find aria_utils)/settings/pid_gain.yaml" ns="/pid_gain"/>
   <node pkg="aria_utils" type="initializer2.py" name="configurator" output="screen"/>
 </group>
</launch>
